deploy/cluster-monitoring-config-non-prod/cluster-monitoring-config.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true 
+    prometheusK8s:
+    # If the location of this config changes,
+    # make sure to adjust https://github.com/openshift/ops-sop/blob/master/v4/howto/o11m_mst_killswitch.md
+    # so that killswitch instructions point to the right file!
+      remoteWrite:
+        - url: http://token-refresher.openshift-monitoring.svc.cluster.local
+          remoteTimeout: 30s
+          writeRelabelConfigs:
+          - sourceLabels: [__name__]
+            action: keep
+            regex: '(cluster_admin_enabled|identity_provider|kube_node_info|kube_node_created|kube_node_role)'
+          queueConfig:
+            capacity: 2500
+            maxShards: 1000
+            minShards: 1
+            maxSamplesPerSend: 500
+            batchSendDeadline: 60s
+            minBackoff: 30ms
+            maxBackoff: 100ms
```

Comment:
@dofinn @arjunrn if we are going to set these parameters for stage in all versions we should do the same in production

Reply:
sure, lets tune stage first tho? try hit staging observatorium with some metrics to see if its client or server at under saturation that causes issues?